week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+  return productPrice + 0.2*productPrice;
```

Comment:
Great. Well done. Another formatting tip for consistency, use spacing between your operators and variables where you can. `return productPrice + 0.2 * productPrice;`